webrender_api/src/image.rs

```diff
@@ -343,7 +343,11 @@ pub trait BlobImageHandler: Send {
 /// A group of rasterization requests to execute synchronously on the scene builder thread.
 pub trait AsyncBlobImageRasterizer : Send {
     /// Rasterize the requests.
```

Comment:
If there's anything interesting to say about the semantics of priority, this is the place to document it, eh?

Reply:
The semantics are a bit specific to working around a surprising behavior in rayon which I hope is temporary. I'll add a hand-wavey comment about this flag telling the rasterizer to schedule work in such a way that low-priority work doesn't block high priority work.